pkg/api/v1/schema.go

```diff
@@ -66,6 +66,10 @@ type ResourceRequirements struct {
 	// Valid resource keys are "memory" and "cpu".
 	// +optional
 	Requests v1.ResourceList `json:"requests,omitempty"`
+	// Limits describes the maximum amount of compute resources allowed.
+	// Valid resource keys are "memory" and "cpu".
```

Comment:
You mention memory as a valid resource, but the rest of the PR seems to specifically address CPU, did I miss something? :)

Reply:
> You mention memory as a valid resource, but the rest of the PR seems to specifically address CPU, did I miss something? :)
You're right, I'll add it in the next PR that handles the memory limits. :)